doc/source/train/examples/lightning/lightning_cola_advanced.ipynb

```diff
@@ -272,13 +515,15 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "## 4. Fine-tune the model with LightningTrainer\n",
```

Comment:
nit: For consistency with other docs we don't need to have numbers in the headers (`4.`) Same applies for the other steps.

Reply:
Ok! Fixed